src/browser/chrome.ts

```diff
@@ -157,8 +191,53 @@ export const openUrl = async ({
         msg.type() === "error" &&
         log(`JS error |`, msg.text().replace(/\n[\w\W]*/, ""), `@ [${url}]`)
     );
+
+    let counterRequestsUnfinished = 0;
+    const onRequestProcessed = async () => {
+      try {
+        await page.evaluate(
+          ({ LAST_RESPONSE_AT_VARIABLE_NAME }) => {
+            const item = Date.now().toString();
+            localStorage.setItem(LAST_RESPONSE_AT_VARIABLE_NAME, item);
+          },
+          { LAST_RESPONSE_AT_VARIABLE_NAME }
+        );
+      } catch (e) {
+        console.error(e);
+      }
+    };
+    const incRequestsUnfinished = async (inc: 1 | -1) => {
+      counterRequestsUnfinished += inc;
+      try {
+        await page.evaluate(
+          ({ counterRequestsUnfinished, COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME }) => {
+            const item = counterRequestsUnfinished.toString();
+            localStorage.setItem(COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME, item);
+          },
+          { counterRequestsUnfinished, COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME }
+        );
+      } catch (e) {
+        console.error(e);
+      }
+      if (inc < 0) {
+        await onRequestProcessed();
```

Comment:
instead of page.evaluate 2 times (which is somewhat heavy and less readable I think), please refactor to use a single page.evaluage 

```ts
await page.evaluate(
          ({ counterRequestsUnfinished, COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME, inc }) => {
            const item = counterRequestsUnfinished.toString();
            localStorage.setItem(COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME, item);
            if (inc < 0) {
              const item2 = Date.now().toString();
              localStorage.setItem(LAST_RESPONSE_AT_VARIABLE_NAME, item2);
            }
          },
          { counterRequestsUnfinished, COUNTER_REQUEST_UNFINISHED_VARIABLE_NAME, inc }
        );
```